### DISCUSSION TITLE
What does `~` mean in a spec?

### DISCUSSION THREAD

**Discussion by marius311** [spack/spack#34674] [Q&A] [answered]
I'm trying to figure out if an e4s environment has Boost compiled with Python bindings, ie if its `boost +python`. If I do `spack find -v boost` I see all the Boost packages that e4s provides are `~python`, but I can't find in the docs what `~` actually means?

**Comment by haampie** [comment, marked as answer]
It means not/false/disabled. See also `spack help --spec`
